junit-jupiter-engine/src/test/java/org/junit/jupiter/api/AssertionsAssertEqualsTests.java

```diff
@@ -556,6 +556,7 @@ void assertEqualsInvokesEqualsMethodForIdenticalObjects() {
 	private static class EqualsThrowsExceptionClass {
 
 		@Override
+		@SuppressWarnings("EqualsHashCode") // purposely testing only that calling .equals() throws an exception
```

Comment:
I added this warning suppression originally because, last time I checked, error-prone failed to compile here because it saw that `hashCode()` was not being overriden as well, as required by the specs of `equals()` and `hashCode()`. However, I can't seem to reproduce this compilation error now for some reason, so I'll see if I can remove it.

FYI, it was reported originally as a result of the rule "EqualsHashCode" (http://errorprone.info/bugpattern/EqualsHashCode).